sys/include/crypto/ciphers.h

```diff
@@ -41,9 +41,7 @@ extern "C" {
  * threedes     needs 24  bytes                           <br>
  * aes          needs CIPHERS_MAX_KEY_SIZE bytes          <br>
  */
-#if defined(MODULE_CRYPTO_3DES)
-    #define CIPHER_MAX_CONTEXT_SIZE 24
-#elif defined(MODULE_CRYPTO_AES)
+#if defined(MODULE_CRYPTO_3DES) || defined(MODULE_CRYPTO_AES)
```

Comment:
Before, `CIPHER_MAX_CONTEXT_SIZE` was `24` for `MODULE_CRYPTO_3DES`, now is `32`.

-> What was the intention behind this?